common/src/main/java/com/wynntils/overlays/minimap/MinimapOverlay.java

```diff
@@ -42,9 +42,12 @@
 import java.util.stream.Stream;
 import net.minecraft.client.gui.Font;
 import net.minecraft.client.renderer.MultiBufferSource;
+import org.joml.Math;
 
 public class MinimapOverlay extends Overlay {
     private static final int DEFAULT_SIZE = 130;
+    private static final float MIN_ZOOM = 0.1f;
+    private static final float MAX_ZOOM = 3;
```

Comment:
Sorry, can you move this to MapRenderer, and use it here and in the map class aswell?